src/applications/benefit-eligibility-questionnaire/components/ShareResultsModal.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+
+import { VaModal } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+const ShareResultsModal = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [showAlert, setShowAlert] = useState(false);
+  const currentUrl = window.location.href;
+  const toggleModal = open => {
+    setIsOpen(open);
+  };
+
+  const handleCopyClick = () => {
+    navigator.clipboard.writeText(currentUrl);
+    setShowAlert(true);
+  };
+
+  return (
+    <div>
+      <va-button
+        id="share-your-results"
+        message-aria-describedby="Share your results"
+        text="Share your results"
+        onClick={() => toggleModal(true)}
+      />
+      <VaModal
+        id="share-results-modal"
+        onCloseEvent={() => {
+          setShowAlert(false);
+          toggleModal(false);
+        }}
+        modalTitle="Save, bookmark, or share your results"
+        initialFocusSelector="#va-modal-title"
+        visible={isOpen}
+        uswds

```

Comment:
The `uswds` prop is no longer necessary for design system web components. They will all default to using the uswds v3 variation now.

Reply:
Ok